exporter/azuredataexplorerexporter/adx_exporter.go

```diff
@@ -181,36 +174,34 @@ func newExporter(config *Config, logger *zap.Logger, telemetrydatatype int) (*ad
 		return nil, err
 	}
 
-	var managedingest *ingest.Managed
-	var queuedingest *ingest.Ingestion
+	var ingestor localIngestor
 
+	ingestoptions := make([]ingest.FileOption, 1)
```

Comment:
A minor thing , the size of this slice can be 1 or 2 depending on if ingest reference is provided